idls/bubblegum.json

```diff
@@ -4365,7 +4365,7 @@
   "metadata": {
     "address": "BGUMAp9Gq7iTEuizy4pqaxsTyUCBK68MDfK752saRPUY",
     "origin": "anchor",
-    "binaryVersion": "0.29.0",
+    "binaryVersion": "0.27.0",
```

Comment:
Is it possible you ran the pnpm:generate with anchor CLI 0.27.0?  Suggest to rerun with 0.29.0 to match the CI build.

Reply:
fixed with clean build (I think 0.27 binary would have been in .crates folder)